index.html

```diff
@@ -11,54 +11,153 @@
 </head>
 
 <body>
-    <header>
-        <nav>
-            <div class="logo">
-                <a href="#">Jonah.</a>
-            </div>
-            <div class="mobile-menu">
-                <a href="#"><i class="fas fa-bars"></i></a>
-            </div>
-        </nav>
-    </header>
-    
     <div class="container">
         <main>
-            <section class="headline">
-                I'm Jonah<br>Glad to see you!
-            </section>
-
-            <section class="body">
-                <p>
-                    I’m a software developer! I can help you build a product feature or website. Look through some
-                    of my work and experience! If you like what you see and have a project you need coded, don’t
-                    hesitate to contact me.
-                </p>
-            </section>
-
-            <section class="social-media">
-                <p>LET'S CONNECT</p>
+            <section class="works">
+                <div class="card">
+                    <div class="snapshot"></div>
+                    <div class="project-title">Tonic</div>
+                    <div class="project-subtitle">
+                        <div class="flex-container-tonic">
+                            <div class="canopy">CANOPY</div>
+                            <div class="dot"></div>
+                            <div>Back End Dev</div>
+                            <div class="dot"></div>
+                            <div>2015</div>
+                        </div>
+                    </div>
+                    <div class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
```

Comment:
- [ ] I strongly suggest using a `<p>` tag for the content (paragraph) instead of `<div>` tag. 👌